### PR TITLE
Basic string operations String slicing and string manipulating

### PR DESCRIPTION
**Pull request by Rohit290699**
Strings in Python:

In Python, strings are sequences of characters, enclosed within either single quotes (' '), double quotes (" ")
Accessing Characters:
You can access individual characters or slices of a string using indexing and slicing

String Methods:
Python provides numerous built-in methods to manipulate strings, such as split(), join(), strip(), replace(), find(), upper(), lower(), capitalize(), title() etc.,

String Concatenation:
You can concatenate strings using the + operator or by simply placing them next to each other

splitting a string:
split() method can be used to split a string into a list of substrings with a specified separator or delimiter. This function returns a list of substrings and accepts the delimiter as an input.

String slicing : "string slicing" can be used to retrieve a specific piece of a string. A new string made consisting of characters from the supplied start index up to the specified end index, but excluding the specified index, is created when you slice a string. Python believes you want to start at the beginning of the string if you don't specify a start index.
Python also believes you want to slice till the end of the string if you don't give an end index. String manipulation can be done precisely thanks to this versatility.
In addition, a step value can be incorporated to skip characters during the slice operation, and negative indices can be utilized to slice in relation to the string
Syntax: Slicing format: string[start : end]

re.findall(): Python's regular expression module (re) contains the re.findall() function, which is used to search over a string and extract all non-overlapping matches of a given pattern. It gives back a list with every match discovered, arranged according to where they are in the string. When it's necessary to extract several instances of a pattern from a given text, this method comes in handy.

In conclusion, regular expressions provide sophisticated pattern-matching capabilities for applications like searching and data extraction, whereas string slicing in Python makes it easier to extract particular chunks of a text by specifying start and end indices. Text processing efficiency and versatility are increased by the re.findall() function, which offers a straightforward method of retrieving every instance of a pattern within a string.

### CODE REVIEW

File: class1.ipynb [hunk 1]
```diff
@@ -0,0 +1,337 @@
+{
```
Comment (nikshepkulli) [inline class1.ipynb:1]: <p>missing header blocks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rohit290699/TBA/pull/6/discussion/#comment-1485215642'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>
